features/HomeTab/FlatOffer/Facilities/Facilities.tsx

```diff
@@ -0,0 +1,51 @@
+import { useContext } from "react";
+import { View, StyleSheet } from "react-native";
+import { IconButton, useTheme, Text } from "react-native-paper";
+
+import SettingsContext from "../../../../contexts/SettingsContext";
+import translations from "../../../../preferences/translations";
+
+const Facilities = () => {
+  const { settings } = useContext(SettingsContext);
+  const theme = useTheme();
+  const items = ["CHAIR", "TV", "WIFI", "BREAKFAST_INCLUDED", "PRIVATE_BATHROOM"];
```

Comment:
We should store facilities in one place e.g. `interfaces/` folder. Now we have discrepancy with this list. Look at: https://github.com/flatly-pw/mobile-app/blob/0263f21673ca4956845931a1b3232ed1b690f3bb/features/HomeTab/AdvancedFilter/AdvancedFilter.tsx#L117-L133